image_to_json.py

```diff
@@ -1,23 +1,65 @@
 #!/usr/bin/env python
-"""
-Opens image and converts to csv data using the PIL library
+"""Converts image to json-like file.
+
+Usage:
+    image_to_json.py (-i FILE) [-o FILE] [-s K] [-f FILTER]
+
+Options:
+    -h --help                   Shows this screen.
+    -i FILE --input=FILE        Input image file.
+    -o FILE --output=FILE       Output json file. [default: image.json]
+    -s K --size=K               Resize image in K. [default: False]
+    -f FILTER --filter=FILTER   Resampling filter, NEAREST, BILINEAR, BICUBIC or LANCZOS. [default: NEAREST]
 """
 
-import Image
-from sys import argv, exit
+
+import os
+import sys
+import getopt
+from docopt import docopt
```

Comment:
Hi Patrick,

I've tried that before, now I'm really keen on google's `yapf`, it eventually can be used with pep8. Do you know this one?

https://github.com/google/yapf

Thanks,

Rodolfo

> Em Jun 10, 2015, às 13:37, Patrick Fuller <notifications@github.com> escreveu:
> 
> In image_to_json.py:
> 
> >  """
> >  
> > -import Image
> > -from sys import argv, exit
> > +
> > +import os
> > +import sys
> > +import getopt
> > +from docopt import docopt
> Do you use any tools for linting / checking style guides? If not, I'd recommend flake8. pip install flake8 and then run with flake8 image_to_json.py. It'll catch some PEP8 style rules too.
> 
> —
> Reply to this email directly or view it on GitHub.
>